### DISCUSSION TITLE
How to disable vimium-c globally for all browser until I press certain shortcut?

### DISCUSSION THREAD

**Discussion by vvkdev** [gdh1995/vimium-c#1089] [Q&A]
I use the extension very rarely, and I would like to activate keyboard control only in these moments.
I tried to search, but I couldn't find the answer (
Thank you!

**Comment by gdh1995** [comment]
For example, if `<f7>` is the shortcut, then the configuration should look like:
![image](https://github.com/gdh1995/vimium-c/assets/5547703/512f65fe-88f1-4ac6-8a64-2d812720cc61)

```
map <f7> openUrl url="vimium://status/toggle-enabled/^%20<f7>"
```

see https://github.com/gdh1995/vimium-c/wiki/Enable-or-Disable-all-frames-by-a-shortcut
